src/main/java/com/avispl/symphony/dal/infrastructure/management/qsc/qsyscore/QSYSCoreAggregatorCommunicator.java

```diff
@@ -590,6 +602,17 @@ public void initQRCCommunicator() throws Exception {
 		qrcCommunicator.init();
 	}
 
+	/**
+	 * Reset socket connection for each polling interval
+	 */
+	public void resetSocketConnection() {
+		if (isFinishedCollectData == 2 && qrcCommunicator != null) {
+			qrcCommunicator.destroyChannel();
```

Comment:
It seems like we're destroying the channel every polling cycle, is this the intention? Wonder if this is done on purpose, because the device is unable to utilize long sessions properly or something?

Reply:
The QSys Core socket closes automatically after 60 seconds (as specified in the API document), however if we let it close automatically that results in unexpected behavior for the adapter, so we close it intentionally.